xmodule/library_sourced_block.py

```diff
@@ -132,28 +245,96 @@ def validate(self):
         validation = super().validate()
         validation = StudioValidation.copy(validation)
 
-        if not self.source_block_ids:
+        if not self.source_library_id:
             validation.set_summary(
                 StudioValidationMessage(
                     StudioValidationMessage.NOT_CONFIGURED,
-                    _("No XBlock has been configured for this component. Use the editor to select the target blocks."),
+                    _("A library has not yet been selected."),
```

Comment:
Minor: I would like to change
`A library has not yet been selected` -> `A library has not been selected yet`